AgentDiscoveries-Backend/README.md

```diff
@@ -66,5 +66,9 @@ Configuration of type `Application`, the main class is
 
 # Unit Tests
```

Comment:
This file now lives in the root directory of the project - hence your merge conflicts :)